_locales/en/messages.json

```diff
@@ -982,7 +982,7 @@
     "description": "Label for when something is turned off"
   },
   "deleteWarning": {
-    "message": "Clicking 'delete' will permanently remove this message from your devices only.",
+    "message": "The message will be deleted from this device only.",
```

Comment:
Just spoke with our Design team and they gave me some updated copy:

> This message will be deleted from this device.

Could you change it to that?

Reply:
Lol, of course, done. That's what I first had written, but then modified it because of the repetition of "this" (see screenshot 🙂)